packages/api/src/db/block.ts

```diff
@@ -102,8 +102,15 @@ export interface ListicleBlock {
 export interface LinkPageBreakBlock {
   type: BlockType.LinkPageBreak
   text: string
+  richText: RichTextNode[]
```

Comment:
Since you need a migration for the `LinePageBreakBlock` i suggest that you replace the `text` field with the `richText`. I don't see any reason why anyone needs both.

Reply:
the `text` is basically the title, the migration was only needed for RTE field and would bring conflicts, so better no field removal / migrations

Reply:
In the `richText` you can have title and text in one. The `richText` field makes the `text` field redundant. And since you're migrating the blocks already it's an easy task to migrate the `text` field to the `richText` field.

Reply:
it breaks the layout and concept of the block. Title and Text (here richText) are two fields